week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item.component.ts

```diff
@@ -0,0 +1,16 @@
+import { div } from '../framework/dom-creators';
+import { Item } from '../models/cart-models';
+
+export function cartItem({ name, amount, unit, price }: Item): HTMLDivElement {
+  const $panelBlock: HTMLDivElement = div('panel-block');
+  const $name: HTMLDivElement = div();
+  $name.textContent = name;
+  const $amount: HTMLDivElement = div('ml-auto');
+  $amount.textContent = amount.toString();
+  const $unit: HTMLDivElement = div('tag');
+  $unit.textContent = unit;
+  const $price: HTMLDivElement = div('ml-4');
+  $price.textContent = `${price.value} ${price.currency}`;
+  $panelBlock.append($name, $amount, $unit, $price);
+  return $panelBlock;
```

Comment:
Widzę u Ciebie częste zapisy np. `: HTMLDivElement`

nie jest to błąd - ale pamiętaj, że w momencie w którym masz dobrze otypowany `div(...` to zapis tego typu
nie jest konieczny.

Jeśli zakładamy że jesteśmy w środku komponentu i bazujemy na referencjach do obiektów `DOM` to 
tutaj nie musimy tego wszędzie dodawać.